### PR TITLE
Add auth check in participant team GET api

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR

- [x] Adds a check to make sure only users who are participant of the team can make `GET` api call to fetch team details

### CODE REVIEW

File: apps/participants/views.py [hunk 1]
```diff
@@ -131,6 +131,14 @@ def participant_team_detail(request, pk):
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
     if request.method == "GET":
+        if not is_user_part_of_participant_team(
+            request.user, participant_team
+        ):
+            response_data = {
+                "error": "You are not a authorized to change team details!"
```
Comment (RishabhJain2018) [inline apps/participants/views.py:138]: `"You are not a authorized to change team details!"` -> `" Sorry, You are not authorized to view team details."`

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Ram81

File: apps/participants/views.py (hunk 1 above)
Comment (RishabhJain2018) [reply apps/participants/views.py:138]: @Ram81 Can you please update this?

**Comment by Ram81** [comment]
@RishabhJain2018 done

**Pull request merged**